homeassistant/components/velbus/config_flow.py

```diff
@@ -144,3 +163,74 @@ async def async_step_discovery_confirm(
             step_id="discovery_confirm",
             description_placeholders={CONF_NAME: self._title},
         )
+
+    async def _validate_vlp_file(self, file_path: str) -> None:
+        """Validate VLP file and raise exception if invalid."""
+        vlpfile = VlpFile(file_path)
+        await vlpfile.read()
+        if len(vlpfile.get()) == 0:
+            raise InvalidVlpFile("no_modules")
+
+    async def async_step_vlp(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Step when user wants to use the VLP file."""
+        if user_input is not None:
+            if CONF_VLP_FILE not in user_input or user_input[CONF_VLP_FILE] == "":
+                # The VLP file is optional, so allow skipping it
+                self._vlp_file = ""
```

Comment:
Should we really use an empty string to mean no file? Wouldn't `None` be better?